docs/getting_started/AppDatamodelUsageWorkflow.md

```diff
@@ -0,0 +1,142 @@
+## App Datamodel Usage Workflow
+
+This workflow will walk you through the steps to interact and work with your applocal datamodel.
+
+* [Pre-requisites](AppDatamodelUsageWorkflow.md#pre-requisites)
+
+* [Usage examples with Nexus Tools](AppDatamodelUsageWorkflow.md#usage-nexus-tools)
+
+
+## Pre-requisites
+1. This workflow requires Datamodel to be initialised and installed before proceeding to further steps. Please follow the below link to configure datamodel
+
+    #### [Datamodel Workflow](DatamodelWorkflow.md)
+
+<!--
+
+1. This workflow requires the App should be initialised and configured before proceeding to further steps. Please follow the below link to configure the App
+
+    #### [App Workflow](AppWorkflow.md)
+-->
+
+<!-- nexus-specific exports
+```
+# store the current directory before we `cd` into the app dir
+export DOCS_INTERNAL_DIR=$PWD/docs/_internal
+```
+-->
+
+
+## Usage examples with Nexus Tools
+
+```
+# Portforward the services Nexus API Gw and GraphQL to access the datamodel objects
+kubectl port-forward svc/nexus-api-gw 5000:80 -n default &
+```
+
+###  Create Leader Object via REST API
+
+**GraphQL query to verify that leader doesn't exist**
```

Comment:
btw why do you use graphql query to check if object is created? 
shouldn't it be like:
Create Leader Object via REST API:
1. Check if object exists using REST API 
2. Create using REST API

Create Manager Object via curl
1. Check if object exits using curl
2. Create object using curl

Reply:
@mkczuk This is intentionally added step to show the GraphQL part